### PR TITLE
Alternative daily weather symbol creation

### PR DESCRIPTION
**Pull request by gavinevans**
Related to https://github.com/metoppv/mo-blue-team/issues/743
Accompanying improver PR: https://github.com/metoppv/improver/pull/2023

Description
This PR adds test data required for the improver PR: . 

The outputs from the new known good outputs are shown below:

### blend_mismatch_inputs
blend_mismatch_inputs

### gridded_input
gridded_input

### gridded_ties
gridded_ties

### single_input
single_input

### spot_input
spot_input

### spot_ties
spot_ties

### CODE REVIEW

**Review by bayliffe — APPROVED** [review]
Tests pass using this data. Plotted outputs look sensible. File sizes limited, though overall this adds quite a lot to the repo given the number of files. Some of these are linked to help reduce volumes.

File: data/weather-symbol-modes/broad_categories.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "wet": [9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30],
+    "dry": [0, 1, 2, 3, 4, 5, 6, 7, 8]
```
Comment (brhooper) [inline data/weather-symbol-modes/broad_categories.json:3]: I don't think we need the nighttime symbols in these lists, as the night -> day symbol unification is done first in the summary symbol calculation.

File: data/weather-symbol-modes/wet_categories.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "extreme_convection": [30, 29, 28, 21, 20, 19],
+    "frozen": [27, 26, 25, 24, 23, 22, 18, 17, 16],
+    "liquid": [15, 14, 13, 12, 11, 10, 9]
```
Comment (brhooper) [inline data/weather-symbol-modes/wet_categories.json:4]: I don't think we need the nighttime symbols in these lists, as the night -> day symbol unification is done first in the summary symbol calculation.

File: data/weather-symbol-modes/intensity_categories.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "rain_shower": [10, 14],
+    "rain": [12, 15],
+    "snow_shower": [23, 26],
+    "snow": [24, 27],
+    "thunder": [29, 30]
```
[the remark this answers is not in the document]
Comment (gavinevans) [reply data/weather-symbol-modes/intensity_categories.json]: I've updated the ordering of the intensity categories json to be as intended. Just to mention, this json wasn't previously actually being used by the acceptance tests, it was just being passed in. As the intensity categories are now being used, that is why the gridded_input/kgo.nc file has been updated in this commit: https://github.com/metoppv/improver_test_data/pull/54/commits/6eaa32f78e216d0fd7ee4a9bc2e936e810a3f864. 

The impact of this change can be seen in this 1D example: [8, 8, 8, 12, 8, 11, 11, 8, 11, 15, 15, 12], where previously without intensity categorisation the result was 11 (the most common code), whilst now the result is 15 (as the 12 and 15 get grouped, and then the 15 is selected as it is first in the intensity categories list).

File: data/weather-symbol-modes/broad_categories.json (hunk 1 above)
Comment (gavinevans) [reply data/weather-symbol-modes/broad_categories.json]: I've removed the night-time codes.

File: data/weather-symbol-modes/wet_categories.json (hunk 2 above)
Comment (gavinevans) [reply data/weather-symbol-modes/wet_categories.json]: I've removed the night-time codes.

**Review by brhooper — APPROVED** [review]
Thanks @gavinevans, this all looks fine to me.

**Pull request merged**